app/src/main/res/values/strings_pin.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="pin">PIN</string>
-</resources>
\ No newline at end of file
+    <string name="setupPin">PIN Setup</string>
+    <string name="phrase1">Let’s create a PIN code to \n secure your ideas.</string>
+    <string name="phrase2">We will ask for it every time you open \n the app.</string>
+    <string name="phrase3">Be attentive, there is no way to restore a \n lost PIN so far.</string>
+    <string name="action_button_next">Next</string>
+    <string name="title">Enter a pin</string>
```

Comment:
let's change the name to `pin_enter_title`. It can be a potential bug because we use it like `R.string.title` so it's not clear it's pin code screen title